fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -2927,6 +2938,22 @@ single_partition_desc ::=
     :}
     ;
 
+multi_partition_desc ::=
+    fixed_multi_partition_key_desc:desc
+        opt_key_value_map:properties
+    {:
+        RESULT = new MultiPartitionDesc(desc, properties);
+    :}
+    ;
+
+fixed_multi_partition_key_desc ::=
+    // FROM (lower) TO (upper) INTERVAL time_interval time_type
+    KW_FROM LPAREN partition_key_list:lower RPAREN KW_TO LPAREN partition_key_list:upper RPAREN KW_INTERVAL INTEGER_LITERAL:time_interval ident:time_type
```

Comment:
use `time_unit` instead of `ident:time_type`

Reply:
I will